Sources/ASN1Decoder/ASN1Decoder.swift

```diff
@@ -53,8 +64,12 @@ final public class ASN1Decoder {
     return result
   }
 
+  /// Read `String` with `UTF8` or `IA5` type
+  /// - Parameters:
+  ///   - pointer: Pointer on start of object. This method move with pointer, in some case you probably need use copy of your pointer
```

Comment:
`Pointer the start of the object. This method moves the pointer.

Reply:
I don't think we need to mention the mutation part here. It's `inout` parameter, it's clear it can mutate the pointer.